website/docs/docs/get-started/develop-in-the-cloud.md

```diff
@@ -1,66 +1,96 @@
+| **Status bar** | This area provides you with useful information about your IDE and project status. You also have additional options like enabling light or dark mode, restarting the IDE, or [recloning your repo](/docs/collaborate/git/version-control-basics).
+| **Dark mode**  | Use dark mode in the Cloud IDE for a great viewing experience in low-light environments. 
+
 
-<table>
-<tr><th>Start up process </th><th>Work retention</th></tr>
-<tr><td>
+**Start up process**
 
 There are three start-up states when using or launching the Cloud IDE:
 
 - Creation start &mdash; This is the state where you are starting the IDE for the first time. You can also view this as a *cold start* (see below), and you can expect this state to take longer because the git repository is being cloned.
 - Cold start &mdash; This is the process of starting a new develop session, which will be available for you for three hours. The environment automatically turns off three hours after the last activity with the rpc server. This includes compile, preview, or any dbt invocation, however, it *does not* include editing and saving a file.
 - Hot start &mdash; This is the state of resuming an existing or active develop session within three hours of the last activity.
 
-
-</td><td>
-
+**Work retention**
 
 The Cloud IDE needs explicit action to save your changes. There are three ways your work is stored:
 
 - Unsaved, local code &mdash; Any code you write is automatically available from your browser’s storage. You can see your changes but will lose them if you switch branches or browsers (another device or browser).
 - Saved but uncommitted code &mdash; When you save a file, the data gets stored in your local storage (EFS storage). If you switch branches but don’t *commit* your saved changes, you will lose your changes.
 - Committed code &mdash; This is stored in the branch with your git provider and you can check out other (remote) branches.
 
-
-</td></tr> </table>
-
-## Set up and access the Cloud IDE
+## Set up Environments
 
 :::info📌
 
 New to dbt? Check out our [Getting Started guide](/docs/get-started/getting-started/overview) to build your first dbt project in the Cloud IDE!
 
 :::
 
-In order to start experiencing the great features of the Cloud IDE, you need to first set up your **Development environment** and **Development credentials.**
+<!-- the link to dbt Cloud environments isn't live just yet, see pr https://github.com/dbt-labs/docs.getdbt.com/pull/2777 -->
+
+In order to start experiencing the great features of the Cloud IDE, you need to first set up a [dbt Cloud environment](docs/collaborate/environments/dbt-cloud-environments#dbt-cloud-environments). Follow the steps in [Development environment](#development-environment) and [Development credentials](#developer-credentials) for more details. 
```

Comment:
hey @dave-connors-3 @nghi-ly and @matthewshaver - i'm wondering if this bit below (dev environment and credentials) should live in Dave's environments page? it differs in that it breaks down steps to set it up but i think combined with Dave's explanation, may be better suited there?

Reply:
@mirnawong1 : good idea! just need to make sure the transition to these sections on dave's page are clear